tests/model_registry/conftest.py

```diff
@@ -229,10 +229,12 @@ def updated_dsc_component_state_scope_class(
     request: FixtureRequest, dsc_resource: DataScienceCluster, admin_client: DynamicClient
 ) -> Generator[DataScienceCluster, Any, Any]:
     original_components = dsc_resource.instance.spec.components
-    with ResourceEditor(patches={dsc_resource: {"spec": {"components": request.param["component_patch"]}}}):
-        for component_name in request.param["component_patch"]:
+    component_patch = request.param["component_patch"]
+
+    with ResourceEditor(patches={dsc_resource: {"spec": {"components": component_patch}}}):
+        for component_name in component_patch:
```

Comment:
You don't need this change, but I have no issues with it being here either. Take this as a FYI and resolve if this is intentional.